.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,13 @@
+contact_links:
+    - name: 📃 Documentation Issue
+      url: https://github.com/PalisadoesFoundation/talawa-docs/issues/new/choose
+      about: This issue tracker is not for Documentation issues. Please file Documentation issues here.
+    - name: 📃 Mobile Issue
+      url: https://github.com/PalisadoesFoundation/talawa/issues/new/choose
+      about: This issue tracker is not for Mobile issues. Please file Mobile issues here.
+    - name: 📃 API Issue
+      url: https://github.com/PalisadoesFoundation/talawa-api/issues/new/choose
+      about: This issue tracker is not for API issues. Please file API issues here.
+    - name: 🤔 Questions and Help
+      url: https://join.slack.com/t/thepalisadoes-dyb6419/shared_invite/zt-2e75w7ups-AVCObKnnHqNzd76CmIRyag
```

Comment:
The slack link changes every 30 days. Use a link to:
- https://github.com/PalisadoesFoundation/talawa